docs/developer-guide/local-config.md

```diff
@@ -187,3 +189,59 @@ Example:<br>
   }
 }
 ```
+
+### projectionDefs configuration
+Custom CRS can be configured here. For example:
+```
+"projectionDefs": [{
+  "code": "EPSG:3003",
+  "def": "+proj=tmerc +lat_0=0 +lon_0=9 +k=0.9996 +x_0=1500000 +y_0=0 +ellps=intl+towgs84=-104.1,-49.1,-9.9,0.971,-2.917,0.714,-11.68 +units=m +no_defs",
+  "extent": [1241482.0019, 973563.1609, 1830078.9331, 5215189.0853],
+  "worldExtent": [6.6500, 8.8000, 12.0000, 47.0500]
+}]
+```
+Explanation of these properties:
+
+- **code** - a code string that will identify the added projection
+- **def** - projection definition in PROJ.4 format
+- **extent** - projected bounds of the projection
+- **worldExtent** - bounds of the projection in WGS84
+
+These parameters for a projection of interest can be looked up on [epsg.io](https://epsg.io)
+
+### CRS Selector configuration
+CRS Selector is a plugin, that is configured in the plugins section. It should look like this:
+```
+"plugins": {
+  ...
+  "desktop": [
+    ...}, {
+      "name": "CRSSelector",
+      "cfg": {
+        "additionalCRS": {
+          "EPSG:3003": {
+            label: "Monte Mario"
+          }
+        },
+        "filterAllowedCRS": [
+          "EPSG:4326",
+          "EPSG:3857"
+        ],
+        "allowedRoles": [
+          "ADMIN"
+        ]
+      }
+    }, {
+  ]
+}
+```
+Configuration parameters are to be placed in the "cfg" object. These parameters are:
+
+- **additionalCRS** - object, that contains additional Coordinate Reference Systems, defined in **projectionDefs**, that are to be available in the CRS Selector, alongside default projections.
+Every additional CRS is a property of **additionalCRS** object. The name of that property is a code of a corresponding projection definition in **projectionDefs**. The value of that property is an object
+with the following properties:
+    - **label** - a string, that will be displayed in the CRS Selector as a name of the projection
```

Comment:
please check indentation here, since it is different from following params like filterAllowedCRS